docs/src/pages/guides/FHIRPerformanceGuide.md

```diff
@@ -2,37 +2,52 @@
-  * [3.1 Concurrency](#31-concurrency)
-  * [3.2 Transaction Timeout](#32-transaction-timeout)
-  * [3.3 Session Affinity](#33-session-affinity)
-  * [3.4 Valud-Id Caches](#34-value-id-caches)
-  * [3.5 Logical Id Generation](#35-logical-id-generation)
-  * [3.6 Compartment Search Optimization](#36-compartment-search-optimization)
-- [4 Database Tuning](#4-database-tuning)
-  * [4.1 PostgreSQL](#41-postgresql)
-  * [4.2 IBM Db2](#42-ibm-db2)
-  * [4.3 Derby](#43-derby)
-- [5 Client Access Scenarios](#5-client-access-scenarios)
-  * [5.1 Search Examples](#51-search-examples)
-- [6 Tools](#6-tools)
-  * [6.1 Making FHIR Requests With curl](#61-making-fhir-requests-with-curl)
-  * [6.2 Making FHIR Requests with IBM FHIR Server Client](#62-making-fhir-requests-with-ibm-fhir-server-client)
-
-# 1 Overview
+**Table of Contents**
+
+- [1. Overview](#1-overview)
+- [2. System Sizing](#2-system-sizing)
+- [3. FHIR Server Configuration](#3-fhir-server-configuration)
+    - [3.1. Concurrency](#31-concurrency)
+        - [3.1.1. Liberty Profile Concurrency](#311-liberty-profile-concurrency)
+        - [3.1.2. Database Max Connections](#312-database-max-connections)
+        - [3.1.3. JEE Datasource Default, Recommended](#313-jee-datasource-default-recommended)
+        - [3.1.4. Proxy Datasource Legacy](#314-proxy-datasource-legacy)
+    - [3.2. Transaction Timeout](#32-transaction-timeout)
+    - [3.3. Session Affinity](#33-session-affinity)
+    - [3.4. Value-Id Caches](#34-value-id-caches)
+    - [3.5. Logical Id Generation](#35-logical-id-generation)
+    - [3.6. Compartment Search Optimization](#36-compartment-search-optimization)
+- [4. Database Tuning](#4-database-tuning)
+    - [4.1. PostgreSQL](#41-postgresql)
+        - [4.1.1. Fillfactor](#411-fillfactor)
+        - [4.1.2. Tuning Auto-vacuum](#412-tuning-auto-vacuum)
+        - [4.1.3. Transaction Id Wraparound](#413-transaction-id-wraparound)
+        - [4.1.4. Vacuum Monitoring](#414-vacuum-monitoring)
+        - [4.1.5. Max Locks](#415-max-locks)
+    - [4.2. IBM Db2](#42-ibm-db2)
+    - [4.3. Derby](#43-derby)
+- [5. Client Access Scenarios](#5-client-access-scenarios)
+    - [5.1. Read](#51-read)
+    - [5.2. Version Read](#52-version-read)
+    - [5.3. History](#53-history)
+    - [5.4. Search Performance](#54-search-performance)
+    - [5.5. Search Examples](#55-search-examples)
+    - [5.6. Tools](#56-tools)
+    - [5.7. Making FHIR Requests With curl](#57-making-fhir-requests-with-curl)
+    - [5.8. Making FHIR Requests with IBM FHIR Server Client](#58-making-fhir-requests-with-ibm-fhir-server-client)
+
+# 1. Overview
 
 This guide describes how to tune IBM FHIR Server and its database to get the best performance. It also describes different FHIR query strategies which may help to work around specific performance issues.
 
 Note: all logical-ids and resources in this guide are examples and do not refer to actual patient data.
 
-# 2 System Sizing
+# 2. System Sizing
```

Comment:
any particular reason for adding the periods after the numbers?  i have no issue with it, but its inconsistent with our user's guide

Reply:
I used a markdown tool to generate it. and it was generated.